integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/SparkIT.java

```diff
@@ -260,6 +260,35 @@ void testListTable() {
     Assertions.assertThrows(NoSuchNamespaceException.class, () -> listTableNames("not_exists_db"));
   }
 
+  @Test
+  public void testAlterTableSetProperty() {
```

Comment:
could you combine set and remove property test into one?

Reply:
fixed